builder/cloudstack/step_prepare_config.go

```diff
@@ -118,6 +127,7 @@ func (s *stepPrepareConfig) Run(state multistep.StateBag) multistep.StepAction {
 
 		types, err := client.GuestOS.ListOsTypes(p)
 		if err != nil {
+			ui.Error(err.Error())
```

Comment:
I think it's much cleaner to replace all the `ui.Error()` lines in this file with just one call to `ui.Error()` right after this line: https://github.com/hashicorp/packer/blob/master/builder/cloudstack/step_prepare_config.go#L136